Explorer/cypress/e2e/poddetails_test.cy.js

```diff
@@ -0,0 +1,33 @@
+describe("ContainerTable", () => {
+  beforeEach(() => {
+    cy.visit("/pods/1"); // Replace with the URL of your application
```

Comment:
this doesn't work anymore after [0118e6c](https://github.com/amosproj/amos2023ss04-kubernetes-inventory-taker/pull/203/commits/0118e6cb05e0e30e564bfc1dcfc4111afe01d993)

a solution could be similar to this approach: https://github.com/amosproj/amos2023ss04-kubernetes-inventory-taker/blob/feat/%2362-pod-details-page/Explorer/cypress/e2e/container_detail_pod_redirect.cy.js

Reply:
Thanks for the pointer on how to solve it, should be fixed now :)